server/server.go

```diff
@@ -1011,6 +1011,25 @@ func (s *Server) Index(w http.ResponseWriter, _ *http.Request) {
 	var lockResults []templates.LockIndexData
 	for id, v := range locks {
 		lockURL, _ := s.Router.Get(LockViewRouteName).URL("id", url.QueryEscape(id))
+
+		queue, _ := s.Locker.GetQueueByLock(v.Project, v.Workspace)
+		var queueIndexDataList []templates.QueueItemIndexData
+		for _, projectLock := range queue {
```

Comment:
This code blocks duplicates the same code added in `server/controllers/locks_controller.go`. Can we turn it into a function and reuse the function in both places?

Reply:
Done with 36046a5